policykit/policyengine/migrations/0001_initial.py

```diff
@@ -487,6 +513,17 @@ class Migration(migrations.Migration):
             },
             bases=('policyengine.constitutionaction',),
         ),
+        migrations.CreateModel(
+            name='PolicykitRecoverCommunityDoc',
+            fields=[
+                ('constitutionaction_ptr', models.OneToOneField(auto_created=True, on_delete=django.db.models.deletion.CASCADE, parent_link=True, primary_key=True, serialize=False, to='policyengine.ConstitutionAction')),
+                ('doc', models.ForeignKey(null=True, on_delete=django.db.models.deletion.SET_NULL, to='policyengine.CommunityDoc')),
+            ],
+            options={
+                'permissions': (('can_execute_policykitrecovercommunitydoc', 'Can execute policykit recover community doc'),),
+            },
+            bases=('policyengine.constitutionaction',),
+        ),
```

Comment:
@rickygv99 we should be adding new migrations instead of updating the existing ones. so, whatever file gets generated from your changes when you run `makemigrations` –– a file prefixed with `0002_`–– should be included in the PR. That way when I or others pull in the changes, we can just apply the new migration file(s). We shouldn't be updating existing migration files, I think, because it will produce errors when trying to `migrate`.

Reply:
Reverted the changes to `0001_initial.py` and created a new migration file: `0002_policy_recovery.py`